src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -45,6 +49,15 @@ public Subsystems() {
 		if (VISION_ENABLED) {
 			visionSubsystem = new VisionSubsystem(poseEstimator);
 		}
+		if (DRIVER_VIS_ENABLED) {
+			if (Hardware.DRIVER_VISION_PATH == null) {
+				DriverStation.reportWarning("No driver vision camera connected!", false);
+			} else {
+				UsbCamera driverVisionCamera =
+						CameraServer.startAutomaticCapture("Driver vision", Hardware.DRIVER_VISION_PATH);
+				driverVisionCamera.setResolution(160, 120);
```

Comment:
Seems pretty low (though will depend on latency and framerates). 
Weren't we able to get 640x480 last year?

Reply:
Haven't gotten drive team feedback yet (Final drive team isn't even finalized, I think?), I'm thinking we'll finalize the resolution during the week of drive team feedback.
I should note the resolutions the camera supports though, so that we know our options when the time comes.